packages/app/src/app/index.js

```diff
@@ -166,6 +166,11 @@ async function initialize() {
         const config = modules[1].config;
 
         overmind = createOvermind(config, {
+          devtools:
+            window.opener && window.opener !== window
```

Comment:
No need to check if `window.opener` exists here

Reply:
We only want to check if there actually is a `window.opener` there. Or it would give "true" when there is no opener, which would cause the wrong result :)